i18n/hi.toml

```diff
@@ -0,0 +1,138 @@
+# i18n strings for the Hindi site.
+
+[deprecation_warning]
+other = "प्रलेखन अब सक्रिय रूप से बनाए नहीं रखा गया है। वर्तमान में आप जिस संस्करण को देख रहे हैं वह एक स्थिर स्नैपशॉट है। अप-टू-डेट प्रलेखन के लिए, देखें | "
+
+[deprecation_file_warning]
+other = "पदावनत" 
+
+[objectives_heading]
+other = "उद्देश्य"
+
+[cleanup_heading]
+other = "सफाई करना"
+
+[prerequisites_heading]
+other = "शुरू करने से पहले"
+
+[whatsnext_heading]
+other = "आगे क्या"
+
+[feedback_heading]
+other = "प्रतिक्रिया"
+
+[feedback_question]
+other = "क्या यह पृष्ठ उपयोगी था?"
+
+[feedback_yes]
+other = "हाँ"
+
+[feedback_no]
+other = "नहीं"
+
+[latest_version]
+other = "नवीनतम संस्करण|"
+
+[version_check_mustbe]
+other = "आपका कुबरनेट्स सर्वर संस्करण होना चाहिए  "
+
+[version_check_mustbeorlater]
+other = "आपका कुबरनेट्स सर्वर संस्करण की तुलना में या बाद में होना चाहिए  "
+
+[version_check_tocheck]
+other = "संस्करण की जांच करने के लिए, प्रवेश करें  "
+
+[caution]
+other = "सावधान:"
+
+[note]
+other = "टिप्पणी:"
+
+[warning]
+other = "चेतावनी:"
+
+[main_read_about]
+other = "के बारे में पढ़ें"
+
+[main_read_more]
+other = "और पढ़ें"
+
+[main_github_invite]
+other = "कोर कुबरनेट्स कोड आधार पर हैकिंग में रुचि रखते हैं?"
```

Comment:
I am finding it difficult to understand this sentence "Interested in hacking on the core Kubernetes code base?"

Since this is not matching up `कोर कुबरनेट्स कोड`